dev-lang/python_bootstrap/python_bootstrap-2.7.17.recipe

```diff
@@ -0,0 +1,102 @@
+SUMMARY="An interpreted, interactive, object-oriented programming language"
+DESCRIPTION="
+Python is a programming language that lets you work more quickly and integrate \
+your systems more effectively. You can learn to use Python and see almost \
+immediate gains in productivity and lower maintenance costs.
+Python runs on Windows, Linux/Unix, Mac OS X, and has been ported to the Java \
+and .NET virtual machines.
+Python is free to use, even for commercial products, because of its \
+OSI-approved open source license.
+"
+HOMEPAGE="https://www.python.org/"
+COPYRIGHT="1990-2018 Python Software Foundation"
+LICENSE="Python"
+REVISION="1"
+SOURCE_URI="https://www.python.org/ftp/python/$portVersion/Python-$portVersion.tar.xz"
+CHECKSUM_SHA256="4d43f033cdbd0aa7b7023c81b0e986fd11e653b5248dac9144d508f11812ba41"
+SOURCE_DIR="Python-$portVersion"
+PATCHES="python_bootstrap-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64 ppc arm arm64 riscv64 sparc m68k"
```

Comment:
Do we want to enable it for all architectures? It may be better to enable it for arm64 (where it was tested) and mark it as untested for the others for now.

Reply:
Good point. We might want to leave everything else as untested.  We'll need to update haikuports cross manifests in the source to update bootstrap packages anyway.. so might as well flag it as untested to keep from updating until needed.

Reply:
It's now ARM64 only.

Reply:
actually.. why not just use ? for each arch?  ex:
```
 ARCHITECTURES="?x86_gcc2 ?x86 ?x86_64 ?ppc ?arm arm64 ?riscv64 ?sparc ?m68k" 
 ```

Reply:
I was told to use ARM64 only. It's not even tested on ARM64. All we know is that it builds. So I prefer to leave it as is.

Reply:
So what is the final verdict for how the ARCHITECTURES line should be. I think this is the only change I need for haikuports.cross.